landing_page/spec/views/microposts/show.html.erb_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe 'microposts/show.html.erb' do
+  before :each do
+    @user = FactoryBot.create(:user)
+    session[:user_id] = @user.id
+    login(@user)
```

Comment:
looks like you are doing the same thing twice here..

Reply:
Good catch!
Resolved in commit 0dcd218